app/views/setting/show.html.erb

```diff
@@ -0,0 +1,28 @@
+<div class="container">
+  <%= form_with url: 'setting/update', method: :post do |s| %>
```

Comment:
you can use here `form_with model`. Because you are updating a model and it will be easier to get and post data with it